dev/cnf/build.gradle

```diff
@@ -428,12 +428,25 @@ task zipGradleBootstrap(type: Zip) {
     }
 }
 
+def getBetaVersion() {
+    def (year, major, minor, month) = "${bnd.libertyRelease}".tokenize('.')
+    int new_month = month.toInteger() + 1
+    if (new_month > 12) {
+        new_month = 1
+        int new_year = year.toInteger() + 1
+        return "" + new_year + "." + major + "." + minor + "." + new_month
```

Comment:
If current release is something like 23.0.0.12, then flow will go into this if statement and return 24.0.0.1?

Reply:
Right